core/src/main/scala/cromwell/core/callcaching/CallCachingMode.scala

```diff
@@ -13,6 +13,8 @@ sealed trait CallCachingMode {
 }
 
 case object CallCachingOff extends CallCachingMode {
+  override val readFromCache = false
+  override val writeToCache = false
```

Comment:
Actually that doesn't directly check internals of `CallCachingMode` behaviour. It does provide a lot of incidental Code Coverage though, so maybe the same thing...